packaging/suse/package_and_push_to_obs.sh

```diff
@@ -10,6 +10,11 @@
 # http://docs.travis-ci.com/user/environment-variables/#Using-Settings
 
 if [ $TRAVIS_PULL_REQUEST == "false" ] && [ $OBS_BRANCH ] && [ $TRAVIS_COMMIT ] && [ $OSC_CREDENTIALS ] && [ $OBS_REPO ] && [ $TRAVIS_BRANCH == $OBS_BRANCH ];then
+  # Clean up the environment
+  rm -r build
+  rm -r *.orig
+  git checkout -- .
```

Comment:
@jordimassaguerpla do you think this is enough the clean up a previous `make_spec.sh` call ?